examples/keras_signle_input_saved_model.rs

```diff
@@ -0,0 +1,45 @@
+use tensorflow::{SessionRunArgs, Graph, SessionOptions, Tensor, SavedModelBundle};
+
+fn main() {
+    // In this file test_in_input is being used while in the python script,
+    // that generates the saved model from Keras model it has a name "test_in".
+    // For multiple inputs _input is not being appended to the op name.
+    let input_op_name = "test_in_input";
+    let output_op_name = "test_out";
+    let save_dir = "examples/keras_signle_input_saved_model";
+    let v: Vec<f32> = vec![0.1, 0.2, 0.3, 0.4, 0.5];
+
+    let tensor = vector_to_tensor(&v);
```

Comment:
As with the other file, this can be simplified to:

```
let tensor = Tensor::from(&[0.1, 0.2, 0.3, 0.4, 0.5][..]);
```

Reply:
Well, it did not work for me because your code is creating tensor of dims:[5], but dim:[1, 5] is expected, so I used the following code:

```rust
let tensor: Tensor<f32> = Tensor::new(&[1, 5])
        .with_values(&[0.1, 0.2, 0.3, 0.4, 0.5])
        .expect("Can't create tensor");
```